nevergrad/optimization/optimizerlib.py

```diff
@@ -367,6 +370,8 @@ class ParametrizedCMA(base.ConfiguredOptimizer):
     ----------
     scale: float
         scale of the search
+    elitist: bool
+        whether we switch to elitist mode
```

Comment:
can you explain in the docstring what is elitist mode?

Reply:
Done.